README.md

```diff
@@ -459,6 +459,16 @@ client.stats(function(err, statsPage) {        // statsPage as PaginatedResult
 client.time(function(err, time) { ... }); // time is in ms since epoch
 ```
 
+### Getting the status of a channel
+
+```javascript
+channel.status(function(err, channelDetails) {
```

Comment:
This example can be moved right next to other `Channel Examples`. Currently, it's under `Ably Time`, which kinda feels not related to channel context.

Reply:
Good point - will do 👍